inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/sink/TubeSink.java

```diff
@@ -121,6 +133,10 @@ public void configure(Context context) {
         // create producer holder
         producerHolder = new TubeProducerHolder(getName(),
                 usedMasterAddr, configManager.getMqClusterConfig());
+        // get statistic configure items
+        maxMonitorCnt = context.getInteger("max-monitor-cnt", 300000);
```

Comment:
move  "max-monitor-cnt" to  ConfigConstants class

Reply:
done